django/cantusdb_project/main_app/templates/edit_volpiano.html

```diff
@@ -0,0 +1,215 @@
+{% extends "base.html" %}
+{% block content %}
+<title>{{ source.title }} | Cantus Manuscript Database</title>
+<script src="/static/js/edit_volpiano.js"></script>
```

Comment:
Since this "edit_volpiano" thing is actually doing more than just editing the volpiano field. Perhaps we should call it `chant_edit`. Could you change the name of JS file and also the name of the template? This is also to stay consistent with the other files in the code base.